registry/paraswap/calldata-AugustusSwapper-v5.json

```diff
@@ -0,0 +1,282 @@
+                        "$ref": "$.display.definitions.amountOutMin"
+                    },
+                    {
+                        "path": "path.[]",
+                        "label": "Array of addresses",
+                        "format": "addressName",
+                        "params": {
+                            "types": ["wallet"]
+                        }
+                    }
+                ],
+                "required": ["amountIn", "amountOutMin", "path"]
+            },
+            "0xf5661034": {
+                "$id": "swapOnUniswapFork",
+                "fields": [
+                    {
+                        "path": "factory",
+                        "$ref": "$.display.definitions.factory"
+                    },
+                    {
+                        "path": "initCode",
+                        "format": "raw",
+                        "label": "init code"
+                    },
+                    {
+                        "path": "amountIn",
+                        "$ref": "$.display.definitions.amountIn"
+                    },
+                    {
+                        "path": "amountOutMin",
+                        "$ref": "$.display.definitions.amountOutMin"
+                    },
+                    {
+                        "path": "path.[]",
+                        "label": "Array of addresses",
+                        "format": "addressName",
+                        "params": {
+                            "types": ["wallet"]
+                        }
+                    }
+                ],
+                "required": ["factory", "initCode", "amountIn", "amountOutMin", "path"]
+            },
+            "0x935fb84b": {
+                "$id": "buyOnUniswap",
+                "fields": [
+                    {
+                        "path": "amountOut",
+                        "$ref": "$.display.definitions.amountOut"
+                    },
+                    {
+                        "path": "amountInMax",
+                        "$ref": "$.display.definitions.amountInMax"
+                    },
+                    {
+                        "path": "path.[]",
+                        "label": "Array of addresses",
+                        "format": "addressName",
+                        "params": {
+                            "types": ["wallet"]
+                        }
+                    }
+                ],
+                "required": ["amountOut", "amountInMax", "path"]
+            },
+            "0xc03786b0": {
+                "$id": "buyOnUniswapFork",
+                "fields": [
+                    {
+                        "path": "factory",
+                        "$ref": "$.display.definitions.factory"
+                    },
+                    {
+                        "path": "initCode",
+                        "format": "raw",
+                        "label": "init code"
+                    },
```

Comment:
Are those displayed by the plugin ? I don't really see how the user can interpret it

Reply:
They are not according tothis method which is called for current function call:
[https://github.com/LedgerHQ/app-plugin-paraswap/blob/72f602251715bf794184721ed0410f0cd631a5d2/src/handle_provide_parameter.c#L48](https://github.com/LedgerHQ/app-plugin-paraswap/blob/72f602251715bf794184721ed0410f0cd631a5d2/src/handle_provide_parameter.c#L48)